theme/templates/page.html

```diff
@@ -3,7 +3,7 @@
 {% block title %}{{ page.title }}{% endblock title %}
 
 {% block content %}
-    <section class="col-12 secondary">
+    <section class="col-12 secondary page">
```

Comment:
I had a different idea but that also works and is simpler compared to what I had in mind. 

But now that we no longer use the `community` class the community page itself has to be updated.